jax/experimental/mosaic/gpu/examples/matmul.py

```diff
@@ -378,7 +378,7 @@ def ref_f(x, y):
         x,
         y,
         dimension_numbers=dimension_numbers,
-        preferred_element_type=jnp.float32,
+        preferred_element_type=out_dtype,
```

Comment:
I think I originally did it to prevent having the baseline use f16 accumulators for 16-bit inputs. Is there some way to enforce that we use 32-bit accumulation in this dot?